readme.md

```diff
@@ -70,56 +70,56 @@ OK, fair.
 ![Amp35](/media/amp_t1.jpg?raw=true)
-The noise parameters were in good agreement with datasheet values, that gave me confidence to go this way. As calculated I need at least 50 MCP6V51s to get under 30nV p-p criteria, so I opted for 100 pieces, in 10x10 matrix, as per idea schematics below
+The noise parameters were in good agreement with the datasheet values, that gave me confidence to go this way. As calculated, I need at least 50 MCP6V51s to get under 30nV p-p criteria, so I opted for 100 pieces, in 10x10 matrix, as per the schematic below
 ![LNA diagram](/media/amp.PNG?raw=true)
-Each 10 first stage amplifiers in noninverting configuration is brought to second stage summing amplifier, and all 10 second stage amplifiers are summed in third stage amplifier. Overall gain of the LNA is 1000.
-Last stage summing amplifier is also used to inject voltage to null offset voltage of LNA. Bart board has DAC and its amplifiers (U119 and U120) for this reason. Second channel of DAC is designed to compensate input current - by injecting current of opposite sign into LNA input. I measured the uncompensated current to be around 2nA and in my current hardware the injection resistor isn't fitted on board. By using this feature, I expect the current all well below 100pA.
-LNA injects ground current into ground net of the circuit, and the current is dependent on LNA input voltage (and therefore LNA output voltage, too). Ground current may cause unwanted voltage shifts between ground potentials of the circuit, especially at high gain operation. To counteract this ground current, amplifier with U121 is setup to source current of opposite polarity into the ground, into areas where LNA resides.
-Bart board is 4-layer, with internal layers being ground nets, for both electrical and thermal reasons.
+Each of the first 10 stage amplifiers in noninverting configuration is brought to a second stage summing amplifier, and all 10 second stage amplifiers are summed into a third stage amplifier. Overall gain of the LNA is 1000.
```

Comment:
Needs a bit of work, maybe something like 
"Each of the first stage gain blocks (with 10 amplifiers in parallel noninverting configuration) is brought to a second stage summing amplifier, and all 10 of the second stage amplifiers are summed into a third, and final, amplifier."